mkt/purchase/webpay.py

```diff
@@ -206,6 +226,20 @@ def simulated_postback(contrib, trans_id):
     return http.HttpResponse(trans_id)
 
 
+def free_postback(request, contrib, trans_id, user_profile):
+    log.info(u'Got free product postback: fulfilling purchase for '
+             'contrib={c}; trans={t}; user={u}'.format(
```

Comment:
annoying, I know, but all concatenated strings need to be unicode literals. This line needs to start with ``u'contrib...``